### DISCUSSION TITLE
Guidelines: Object Naming

### DISCUSSION THREAD

**Discussion by waldo1001** [microsoft/alguidelines#64] [BC Best Practices]

**Comment by christianbraeunlich** [comment]
MS Docs on _Object Naming_:

- https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/compliance/apptest-bestpracticesforalcode#examples-of-object-naming
- https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/compliance/apptest-bestpracticesforalcode#object-naming

MS Docs on _Prefix and Suffix_: https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/compliance/apptest-prefix-suffix

---

I see some space for improvements here.

We should provide some information on how to set the pre-/suffix for object naming automatically by using https://marketplace.visualstudio.com/items?itemName=waldo.crs-al-language-extension
